Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM node:carbon
```

Comment:
Tag this -- it's going to break if we ever use an API that is not backward compatible. Also use the `slim` image from the [node base image](https://hub.docker.com/r/_/node/).